modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -30,14 +32,16 @@ Source environment prerequisites::
 ** The VM must not be configured for USB devices.
 ** The watchdog model must not be `diag288`.
 
-Target environment prerequisites::
+{VirtProductName} environment::
```

Comment:
Nit: Looking at the build, it doesn't look like description list markup without indentation works particularly well here.

Reply:
You're right. I reverted some changes but forgot to revert these. Will fix.

Reply:
Done